tools/airgap_linter.py

```diff
@@ -127,10 +127,10 @@ def validate_images(framework_directory):
         for line in lines:
             line = line.strip()
             if "image:" in line:
-                image_matcher = re.compile("image:\s?(.*)$", re.IGNORECASE)
+                image_matcher = re.compile(r"image:\s?(.*)$", re.IGNORECASE)
                 match = image_matcher.match(line)
                 image_path = match.group(1)
-                env_var_matcher = re.compile("[\"]?\{\{[A-Z0-9_]*\}\}[\"]?")
+                env_var_matcher = re.compile(r'["]?\{\{[A-Z0-9_]*\}\}["]?')
```

Comment:
@porridge I hope that your concern here is now addressed.
@mhrabovcin I hope that your linters still run.

Reply:
```
>>> "[\"]?\{\{[A-Z0-9_]*\}\}[\"]?" == r'["]?\{\{[A-Z0-9_]*\}\}["]?'
True
>>> 
```
I'm happy!